### DISCUSSION TITLE
Removing/hiding field Project, Activity and Billable for user

### DISCUSSION THREAD

**Discussion by beriknl** [kimai/kimai#2632] [Questions & Answers]
Is it possible, in the time registration window, to hide the fields for the user input? I just want them to capture the hours/dates only. 

Would this be possible? 
So just keep From, To, Duration, Label and Description.

For those encountering the same issue, I used the following work-around:
In file templates/timesheet/layout-edit.html.twig, go to line 119. Here start a Javascript. 

I've added the following Javascript:
```
$( document ).ready(function() {
$("#timesheet_edit_form_billable").parent().hide();
$("#timesheet_edit_form_project").val("1").change();
$("#timesheet_edit_form_project").parent().hide();
});
```
It set's the project to value 1 and hides both billable and the project field. The 1 represents project number (you can identify from the source in the html). Off course not ideal, since it's client side, but it's a workaround. Given the activity list is loaded through an different ajax call to the api, and seems a different list, this logic does not apply there and is still required to be selected. 

After run command bin/console kimai:reload --env=prod to re-generate.

**Comment by kevinpapst** [comment]
Answered a hundred times before: no.
If you want to know more, search the closed issues.

**Reply by kevinpapst** [reply]
Your solution might do the trick, even though I cannot support that.

I would propose a very slim JS frontend instead, that uses the Kimai API to start and stop entries.
Then admins can use the full application and the "simple users" have the full customized version.
